pallets/msa/src/lib.rs

```diff
@@ -748,7 +750,7 @@ pub mod pallet {
 		/// - [`DelegationNotFound`](Error::DelegationNotFound) - If there is not delegation relationship between Origin and Delegator or Origin and Delegator are the same.
 		/// - [`SchemaNotGranted`](Error::SchemaNotGranted) - If attempting to revoke a schema that has not previously been granted.
 		///
-		#[pallet::weight(T::WeightInfo::revoke_schema_permissions(20_000))]
+		#[pallet::weight(T::WeightInfo::revoke_schema_permissions(20_000))] // Number of delegations is not part of the extrinsic complexity
```

Comment:
@enddynayn

Reply:
This one is being addressed in this outstanding PR:
https://github.com/LibertyDSNP/frequency/pull/720